CONTRIBUTING.md

```diff
@@ -300,3 +300,18 @@ Run the reports that are defined in `phpbench.json` via the command line,
 for example:
 
     vendor/bin/phpbench run --report=ast
+
+## Local setup
+It is also possible to setup the test environment without usage of the docker and make files.
+You would need the following:
+
+- PHP (at least 7.2)
+- Composer
+- A Mysql 8 schema
```

Comment:
Can you clarify?

Reply:
I could prob. write it a bit better yeah :)